system/lib/libc/musl/src/time/clock_settime.c

```diff
@@ -1,7 +1,16 @@
 #include <time.h>
 #include "syscall.h"
+#ifdef __EMSCRIPTEN__
+#include <errno.h>
+#endif
```

Comment:
This seems unnecessary.

Reply:
We do write to errno below, and I verified this fails to compile without it (so it's not included in the other headers).